ts/api/api.ts

```diff
@@ -325,6 +331,14 @@ class OpenYoloApiImpl implements OpenYoloApi {
         this.areTimeoutsDisabled ? undefined : DEFAULT_TIMEOUTS.proxyLogin;
     return request.dispatch(credential, timeoutMs);
   }
+
+  async cancelLastOperation() {
```

Comment:
You should do like all the other requests, i.e. have one using the browser (i.e. in this case, raise an error probably as you cannot cancel) and one that uses the IFrame.